core/src/wsv.rs

```diff
@@ -523,43 +523,9 @@ impl WorldStateView {
         )
     }
 
-    /// The same as [`Self::modify_world_multiple_events`] except closure `f` returns a single [`WorldEvent`].
-    ///
-    /// # Errors
-    /// Forward errors from [`Self::modify_world_multiple_events`]
-    pub fn modify_world(
```

Comment:
Reason why i removed this method is because it was build upon assumption on interior-mutability (and could trigger deadlock), without it is not so convenient to use.

Reply:
I agree